test/integration/openshift_health_checker/teardown_container.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: stop test container
+  delegate_to: localhost
```

Comment:
It's way better than the original approach which was to explicitly delete by name (more opportunities for copy-pasta)!

The other, more intuitive approach would be to run a play against localhost and iterate `hostvars` for the list of hostnames to cleanup. However, in order to recover from test exceptions, this cleanup has to be done in a task on the same block as the test tasks. So it has to happen "on" the host (though via connection to localhost). It does seem a little magical so it's probably worth a comment to explain the magic.